docs/zh_CN/Assessor/MedianstopAssessor.md

```diff
@@ -2,4 +2,4 @@
 
 ## Median Stop
 
-Medianstop 是一种简单的提前终止 Trial 的策略，可参考[论文](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46180.pdf)。 如果 Trial X 的在步骤 S 的最好目标值比所有已完成 Trial 的步骤 S 的中位数值明显要低，这个 Trial 就会被提前停止。
\ No newline at end of file
+Medianstop 是一种简单的提前终止 Trial 的策略，可参考[论文](https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/46180.pdf)。 如果 Trial X 在步骤 S 的最好目标值低于所有已完成 Trial 前S个步骤目标平均值的中位数，这个 Trial 就会被提前停止。
```

Comment:
thanks for your fix, but could you revert this change, because we have a dedicated pipeline for the update of chinese doc

Reply:
we will update this part later :)

Reply:
@QuanluZhang Done.